examples/notifications-solana/pages/index.tsx

```diff
@@ -66,8 +66,12 @@ function AuthedHome() {
             notifications={[
               {
                 name: 'Welcome Message',
-                detail:
-                  'Welcome message that is sent on first subscription',
+                detail: 'Welcome message that is sent on first subscription',
+                renderAdditional: () => (
```

Comment:
let's add optional argument 'state' to the example. something like

```
renderAdditional: (state: boolean) => (
    <div className="text-xs text-neutral-400">
        Additional information about the welcome message (now {state ? 'turned on' : 'turned off'})
    </div>
    )
```

Reply:
Since these are local notifications, there is no toggle, which means there is no state.

I agree that we need an example for remote notifications, but we don't have any for now.